src/sidebar/components/Annotation/AnnotationHeader.tsx

```diff
@@ -110,7 +110,7 @@ function AnnotationHeader({
       <div className="flex gap-x-1 items-baseline flex-wrap-reverse">
         {isPrivate(annotation.permissions) && !isEditing && (
           <LockIcon
-            className="text-tiny w-em h-em"
+            className="w-[10px] h-[10px]"
```

Comment:
Here and the following icon: We want these icons to be 10px square. This really has nothing to do with text sizing...